Bottom: ab31537b275d4dd0c6a20822b458338f6f5b6bd2
Top:    41680838ff5d6d639ac06ed409e8cd995dd14a08
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 16:06:45 +0000

Refresh of fix-a-few-random-places-where

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 405f9ab..4553186 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -251,6 +251,9 @@ object3d_from_board_outline (void)
 
   board_outline = board_outline_poly (true);
 
+  if (board_outline == NULL)
+    return NULL;
+
   /* Loop over all board outline pieces */
   pa = board_outline;
   do {
